packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -454,17 +479,45 @@ export default {
         }
     }
 
+    &.c-megaModal-modeRTL {
+        .c-megaModal-closeBtn {
+            right: f.spacing(d);
+            left: auto;
+            transform: scaleX(-1);
+        }
+    }
+
+    &.c-megaModal--hasBackBtn {
+        .c-megaModal-closeBtn {
+            left: f.spacing(d);
+        }
+    }
```

Comment:
[This](https://rtlstyling.com/posts/rtl-styling#css-logical-properties) is also a good reference

Reply:
Yeah fully appreciate that. In my opinion, I think we should be relying on the consuming application but maybe this is a direction we should figure out as a team as it will also inform the WC work. I can set up a chat with the team, that might be easier than a PR thread :)